src/components/header/header.scss

```diff
@@ -0,0 +1,60 @@
+$primary-color: rgb(27, 41, 48);
```

Comment:
This is really a matter of preference, but I find it's easier to store all variables in a partial, then import the partial. No need to hunt down the specific variable, and it allows you to handle globals easier.